extensions/ql-vscode/src/remote-queries/view/RemoteQueries.tsx

```diff
@@ -39,74 +36,64 @@ const AnalysisResult = (props: AnalysisResult) => (
   </span>
 );
 
-export function RemoteQueries(props: Props): JSX.Element {
-  return <div className="vscode-codeql__remote-queries-view">
-    <h1 className="vscode-codeql__query-title">{props.queryTitle}</h1>
+export function RemoteQueries(_: Record<string, never>): JSX.Element {
```

Comment:
nit: I _think_ this will work too:

```suggestion
export function RemoteQueries(): JSX.Element {
```